### ISSUE TITLE
Motor Voltage Compensation

### ISSUE THREAD

**Issue by chmoore889** [FIRSTTeam6423/2020RobotCode#9]
Add voltage compensation to motor controllers.

**Comment by Team6423** [comment]
Yes we can put on voltage compensation mode. The command is motorName.enableVoltageCompensation(true) - true is on, false is off

**Comment by Team6423** [comment]
But if you put voltage compensation on every motor it will end up doing nothing, so you have to choose what is most important to not brown out. Also charging your batteries will be very helpful in not using the function frequently.
